gsa/src/web/components/powerfilter/filtersearchgroup.js

```diff
@@ -28,25 +31,33 @@ import PropTypes from '../../utils/proptypes.js';
 import FormGroup from '../form/formgroup.js';
 import TextField from '../form/textfield.js';
 
-const OwnerGroup = ({owner, filter, onChange, name = 'owner'}) => {
-  if (!isDefined(owner) && isDefined(filter)) {
-    owner = filter.get('owner');
+const FILTER_TITLE = {
+  comment: 'Comment',
```

Comment:
Personally I would extend FilterSearchGroup with an additional title prop which allows the parent component to set the title instead of adding a map.

Reply:
Done. There are six new commits, but all of them come from fixing conflicts (between local and remote branch) that resulted from removing things like FILTER_TITLE and adding a "title" prop to every search field on the results filterdialog page. Fundamentally I didn't change that much.